lib/test/t_NAIS_std.cxx

```diff
@@ -89,6 +89,6 @@ int main()
   algoNais.run();
 
   const NAISResult resultNais(algoNais.getResult());
-  OT::Test::assert_almost_equal(resultNais.getProbabilityEstimate(), 0.00227031);
+  OT::Test::assert_almost_equal(resultNais.getProbabilityEstimate(), 0.00215688);
```

Comment:
It would be better to manually specify the tolerances IMO:

`assert_almost_equal(tested, reference, relative_tolerance, absolute_tolerance)`

I never know the default values!

Reply:
by default rtol = 1.0e-5, atol = 1.0e-8; these are fine in many cases
seems everyone approved and and there's only this minor remark left then lets merge it